client/components/inline-support-link/index.jsx

```diff
@@ -31,6 +31,8 @@ class InlineSupportLink extends Component {
 		supportLink: PropTypes.string,
 		showText: PropTypes.bool,
 		text: PropTypes.string,
+		showHero: PropTypes.bool,
```

Comment:
Is there a reason for two props here? Could we use just the `heroImage` prop, and display when not empty?

Reply:
@kwight I was following the pattern already used in the component with showText and text. Is there a value to consistency there, or should I swap it to use one prop anyways?

Reply:
If @blackjackkent lands https://github.com/Automattic/wp-calypso/pull/44473

We can avoid the extra prop here in favor of using children. Eg:
```jsx
<InlineSupportLink>
    <img src={ heroImage } />
</InlineSupportLink>
```